morpheushelper/cogs/invites.py

```diff
@@ -17,6 +17,7 @@
 from models.allowed_invite import InviteLog, AllowedInvite
 from permissions import Permission
 from util import send_to_changelog, get_prefix
+import logging
```

Comment:
perhaps you should consider using the `getLogger` method of logging to get a specific logger and also create a logging config.
```python
logger = logging.getLogger(<name>)
```